packages/ckeditor5-markdown-gfm/tests/gfmdataprocessor/code.js

```diff
@@ -23,18 +23,6 @@ describe( 'MarkdownGfmDataProcessor', () => {
 			);
 		} );
 
-		it( 'should process spaces inside inline code', () => {
-			testDataProcessor(
-				'regular text and` inline code`',
-
-				'<p>regular text and<code> inline code</code></p>',
-
-				// When converting back it will be normalized and spaces
-				// at the beginning of inline code will be removed.
-				'regular text and `inline code`'
```

Comment:
I don't see a reason why this should be “normalized” like this. It seems invalid.